docs/cloud/rahti/storage/index.md

```diff
@@ -23,3 +23,5 @@ You can find additional information on the [Ephemeral storage](/cloud/rahti/stor
 In case you need to store large volumes of data, or you need that your data can be easily accessed over the Internet, e.g., using URLs, object storage is the storage solution you are looking for. By using object storage, you are free to create and delete your Pods as the object storage is not tied to any Pod. Moreover, your data is replicated, thus object storage represents a reliable long-term storage option.
 
 At CSC, we offer Allas as our object storage solution. You can find additional information on the [Allas page](../../../data/Allas/index.md).
+
+You can find additional examples on how to backup to Allas [here](objectstorage).
```

Comment:
Link does not work, at least in the preview site.

Reply:
I changed the link to make it as the same format as the other from this article.
With the preview, it doesn't work